xbmc/pvr/addons/PVRClients.h

```diff
@@ -155,12 +155,21 @@ namespace PVR
     bool HasConnectedClients(void) const;
 
     /*!
-     * @brief Get the friendly name for the client with the given id.
+     * @brief Get the (friendly) name for the client with the given id.
      * @param iClientId The id of the client.
-     * @param strName The friendly name of the client or an empty string when it wasn't found.
+     * @param strName The (friendly) name of the client or an empty string when it wasn't found.
+     * @param bFriendly return friendly vs. plain addon name
      * @return True if the client was found, false otherwise.
      */
-    bool GetClientName(int iClientId, std::string &strName) const;
+    bool GetClientName(int iClientId, std::string &strName, bool bFriendly = true) const;
```

Comment:
in that case we should add a new method `GetFriendlyName()` instead of passing a bool in. much more friendly :smile: